packages/edit-site/src/components/global-styles/font-library-modal/index.js

```diff
@@ -54,22 +57,39 @@ function FontLibraryModal( {
 			isFullScreen
 			className="font-library-modal"
 		>
-			<TabPanel
-				className="font-library-modal__tab-panel"
-				initialTabName={ initialTabName }
-				tabs={ tabs }
-			>
-				{ ( tab ) => {
-					switch ( tab.name ) {
-						case 'upload-fonts':
-							return <UploadFonts />;
-						case 'installed-fonts':
-							return <InstalledFonts />;
-						default:
-							return <FontCollection id={ tab.name } />;
-					}
-				} }
-			</TabPanel>
+			<div className="font-library-modal__tabs">
+				<Tabs initialTabId={ initialTabId }>
+					<Tabs.TabList>
+						{ tabs.map( ( { id, title } ) => (
+							<Tabs.Tab key={ id } tabId={ id }>
+								{ title }
+							</Tabs.Tab>
+						) ) }
+					</Tabs.TabList>
+					{ tabs.map( ( { id } ) => {
+						let contents;
+						switch ( id ) {
+							case 'upload-fonts':
+								contents = <UploadFonts />;
+								break;
+							case 'installed-fonts':
+								contents = <InstalledFonts />;
+								break;
+							default:
+								contents = <FontCollection id={ id } />;
+						}
+						return (
+							<Tabs.TabPanel
+								key={ id }
+								tabId={ id }
+								focusable="false"
```

Comment:
Tell me this file doesn't have TypeScript checks without telling me that this file doesn't have TypeScipt type checks

```suggestion
								focusable={ false }
```